app/app_client.go

```diff
@@ -0,0 +1,1241 @@
+	_, err := c.client.DeleteOrganizationMember(ctx, &pb.DeleteOrganizationMemberRequest{
+		OrganizationId: orgID,
+		UserId:         userID,
+	})
+	return err
+}
+
+// DeleteOrganizationInvite deletes an organization invite.
+func (c *AppClient) DeleteOrganizationInvite(ctx context.Context, orgID, email string) error {
+	_, err := c.client.DeleteOrganizationInvite(ctx, &pb.DeleteOrganizationInviteRequest{
+		OrganizationId: orgID,
+		Email:          email,
+	})
+	return err
+}
+
+// ResendOrganizationInvite resends an organization invite.
+func (c *AppClient) ResendOrganizationInvite(ctx context.Context, orgID, email string) (*OrganizationInvite, error) {
+	resp, err := c.client.ResendOrganizationInvite(ctx, &pb.ResendOrganizationInviteRequest{
+		OrganizationId: orgID,
+		Email:          email,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return organizationInviteFromProto(resp.Invite), nil
+}
+
+// EnableBillingService enables a billing service to an address in an organization.
+func (c *AppClient) EnableBillingService(ctx context.Context, orgID string, billingAddress *BillingAddress) error {
+	_, err := c.client.EnableBillingService(ctx, &pb.EnableBillingServiceRequest{
+		OrgId:          orgID,
+		BillingAddress: billingAddressToProto(billingAddress),
+	})
+	return err
+}
+
+// DisableBillingService disables the billing service for an organization.
+func (c *AppClient) DisableBillingService(ctx context.Context, orgID string) error {
+	_, err := c.client.DisableBillingService(ctx, &pb.DisableBillingServiceRequest{
+		OrgId: orgID,
+	})
+	return err
+}
+
+// UpdateBillingService updates the billing service of an organization.
+func (c *AppClient) UpdateBillingService(
+	ctx context.Context, orgID string, billingAddress *BillingAddress, billingSupportEmail string,
+) error {
+	_, err := c.client.UpdateBillingService(ctx, &pb.UpdateBillingServiceRequest{
+		OrgId:               orgID,
+		BillingAddress:      billingAddressToProto(billingAddress),
+		BillingSupportEmail: billingSupportEmail,
+	})
+	return err
+}
+
+// OrganizationSetSupportEmail sets an organization's support email.
+func (c *AppClient) OrganizationSetSupportEmail(ctx context.Context, orgID, email string) error {
+	_, err := c.client.OrganizationSetSupportEmail(ctx, &pb.OrganizationSetSupportEmailRequest{
+		OrgId: orgID,
+		Email: email,
+	})
+	return err
+}
+
+// OrganizationGetSupportEmail gets an organization's support email.
+func (c *AppClient) OrganizationGetSupportEmail(ctx context.Context, orgID string) (string, error) {
+	resp, err := c.client.OrganizationGetSupportEmail(ctx, &pb.OrganizationGetSupportEmailRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return "", err
+	}
+	return resp.Email, nil
+}
+
+// CreateLocationOptions contains optional parameters for CreateLocation.
+type CreateLocationOptions struct {
+	ParentLocationID *string
+}
+
+// CreateLocation creates a location.
```

Comment:
it is a little unclear what the parameters here are being used for, maybe add something like:
 ```
// CreateLocation creates a new location with the specified name under the given orgID. 
// Optionally, the location can be placed under a new parent location specified in the options.
```